### PR TITLE
98212 LTC json to xml mapping for migrate existing licence

### PR DESCRIPTION
**Pull request by sakshi15525**
1. Json to xml mapping for migrate existing licence events.
2. Changed attribute 'upn' to 'ecdisManufacturerName' for ros events.
3. Added logging.
4. Added Unit test cases.
5. Added functional tests.

### CODE REVIEW

**Review by pravinvarade10801 — APPROVED** [review]
LGTM

File: tests/UKHO.ERPFacade.EventAggregation.WebJob.Tests/Helpers/RecordOfSaleSapMessageBuilderTests.cs [hunk 1]
```diff
@@ -82,15 +82,16 @@ public class RecordOfSaleSapMessageBuilderTests
 ";
 
         #region Data
-
         private readonly string jsonForNewLicence1 = @"{""specversion"": ""1.0"",""type"": ""uk.gov.ukho.shop.recordOfSale.v1"",""source"": ""https://uk.gov.ukho.shop"",""id"": ""e744fa37-0c9f-4795-adc9-7f42ad8f11c1"",""time"": ""2023-07-20T10:40:00.000000Z"",""subject"": ""releasability set changes holdings Record of Sale"",""datacontenttype"": ""application/json"",""data"": {""correlationId"": ""123-abc-456-xyz-333"",""relatedEvents"": [""e744fa37-0c9f-4795-adc9-7f42ad8f11c1"", ""e744fa37-0c9f-4795-adc9-7f42ad8f1234""],""recordsOfSale"":
```
Comment (Vishal14583) [inline tests/UKHO.ERPFacade.EventAggregation.WebJob.Tests/Helpers/RecordOfSaleSapMessageBuilderTests.cs:85]: Rename all variables to meaningful name.

**Review by Vishal14583 — CHANGES_REQUESTED** [review]
Rename variables to meaningful names.

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Review by kishorambare — APPROVED** [review]
LGTM.

**Review by jrippington — APPROVED** [review]
Looks good

**Pull request merged**